recipes/salib/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: salib
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/SALib/SALib
+  git_tag: v{{ version }}
```

Comment:
You shouldn't use git here because it takes extra time and resources.  Please use a tarball, which you can get from github.